kwok/apis/apis.go

```diff
@@ -34,6 +32,6 @@ var (
 	//go:embed crds/karpenter.kwok.sh_kwoknodeclasses.yaml
 	KWOKNodeClassCRD []byte
 	CRDs             = []*v1.CustomResourceDefinition{
-		lo.Must(functional.Unmarshal[v1.CustomResourceDefinition](KWOKNodeClassCRD)),
+		object.Unmarshal[v1.CustomResourceDefinition](KWOKNodeClassCRD),
```

Comment:
no must anymore? or does object do a must?

Reply:
Object does a must for us